java/squeek/veganoption/integration/waila/ProviderJutePlant.java

```diff
@@ -34,9 +35,9 @@ public ItemStack getWailaStack(IWailaDataAccessor accessor, IWailaConfigHandler
 		{
 			float growthValue = ((BlockJutePlant) accessor.getBlock()).getGrowthPercent(accessor.getWorld(), accessor.getPosition(), accessor.getBlockState()) * 100.0F;
 			if (growthValue < 100)
-				toolTip.add(String.format("%s : %.0f %%", I18n.format("hud.msg.growth"), growthValue));
+				toolTip.add(LangHelper.translate("waila.growth", (int) growthValue));
```

Comment:
Last I checked, Minecraft just uses standard string formatting, so it shouldn't be any different. What was showing up when you had %.0f in the lang file?